manifest.toml

```diff
@@ -18,13 +18,12 @@ code = "https://github.com/Chocobozzz/PeerTube"
 cpe = "cpe:2.3:a:framasoft:peertube"
 
 [integration]
-yunohost = ">= 11.2.30"
```

Comment:
Why forcing bookworm, the app still works on bullseye

Reply:
IMO, all the apps that use helpers v2.1 should have YNH 12 as a minimum requirement. Helpers V2.1 are getting bug fixes that are not ported to the YNH 11 branch.